pkg/webui/console/store/middleware/logics/lib.js

```diff
@@ -89,7 +94,20 @@ const createRequestLogic = function(
         } else {
           // Otherwise, dispatch the fail action and report it to Sentry.
           if (isUnknown(e) || isInvalidArgumentError(e)) {
```

Comment:
What was actually the reason for distinguishing invalid argument errors?

Reply:
the reason is that there is a lot of errors that are part of the regular flow, `404` for example always occurs when creating new gateways which haven't been connected. We don't want this to consume our sentry subscription.

Reply:
I get that for 404s, but invalid argument is not a 404?

Reply:
yeah I think we decided that `invalidArgumentError` is the one that is not part of any regular flow, so it is good to report it to Sentry. The rest of the errors (404, 401, 403...) is not reported to Sentry.